src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -82,6 +82,7 @@ export const emptyHeaderSx = {
  *
  */
 export function EmbeddedPositionSearchPanel({
+                                                initArgs={},
```

Comment:
Why pass `initArgs` instead of getting them from context?

Reply:
I wanted to leave the option here to control if `EmbeddedPositionSearchPanel` supports init args. I think there are be cases (with DCE) when it should not so I decided to just pass it.